server/queries/queries.js

```diff
@@ -0,0 +1,104 @@
+const { Client } = require('pg');
+const Promise = require('promise');
+
+let pool;
+
+if (process.env.DATABASE_URL) {
+    pool = new Client({
+        connectionString: process.env.DATABASE_URL
+    });
+} else {
+    pool = new Client({
+        user: process.env.USER,
+        host: process.env.HOST,
+        database: 'telltales',
+        password: process.env.PASSWORD,
+        port: process.env.PORT
+    });
+}
+
+pool.connect((err) => {
+    if (err) {
+        console.error('Connection error', err.stack);
+    } else {
+        console.log("Connected");
+    }
+});
+
+// Create the table if it doesn't already exist
+/* eslint max-len: ["error", { "code": 200 }] */
+const createTableQuery = 'CREATE TABLE IF NOT EXISTS users ( id SERIAL PRIMARY KEY, name VARCHAR(15) NOT NULL, score INT );'
+pool.query(createTableQuery, 
+    (err) => {
+    if (err) {
+        console.log(err);
+    } else {
+        console.log("Created table users");
+    }
+});
+
+// Add a user to the table
+module.exports.add = (username) => {
```

Comment:
`module.exports.add = username => {` and in a bunch of other places in this file